bot/persistence/resources/init_db.sql

```diff
@@ -2,3 +2,8 @@ CREATE TABLE IF NOT EXISTS configs(
     config_key varchar PRIMARY KEY,
     val varchar NOT NULL
 );
+
+CREATE TABLE IF NOT EXISTS Modmail(
```

Comment:
We should be consistent with column and table naming strategies.
I prefer lowercase column names (single words) and lower snake_case table names. But feel free to convince me otherwise. Just make it consistent

Reply:
Singular and PascalCase for **_everything_**.

Reply:
Your strong and uncountable argumentation chain convince me :P